src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -195,8 +195,11 @@ class EmojiPicker extends React.Component<IProps, IState> {
     };
 
     private emojiMatchesFilter = (emoji: IEmoji, filter: string): boolean => {
-        return emoji.annotation.toLowerCase().includes(filter) ||
-            emoji.emoticon?.toLowerCase().includes(filter) ||
+        return emoji.label.toLowerCase().includes(filter) ||
+            (Array.isArray(emoji.emoticon)
+                ? emoji.emoticon.some((x) => x.includes(filter))
+                : emoji.emoticon.includes(filter)
```

Comment:
![image](https://user-images.githubusercontent.com/4224639/145676793-81e1a15a-af11-4e5c-9edf-f46c5f1a60b1.png)

```suggestion
                : emoji.emoticon?.includes(filter)
```

Reply:
Thanks for the catch!